### ISSUE TITLE
Multiple rules within a policy are not returned

### ISSUE THREAD

**Issue by migster8** [Turall/OPA-python-client#17]
Hi @Turall 

For cases where there is more than one rule within a policy, the following functions don't appear to work as expected:
 - check_permission
 - get_policies_info

In both cases, it will only return one rule. It seems the issue lies with an if statement, e.g.:

```python
# problem code
for rule in result.get('ast').get('rules'):
  if not rule.get('default'):
      continue
  if rule.get('head').get('name') == rule_name:
    permission_url += '/' + rule.get('head').get('name')
    find = True
```
As additional rules are not considered default, it will skip them completely (at least that's what I think). As a proposed solution, suggest the following:

```python
# solution code
rules = [rule.get("head").get("name") for rule in result.get("ast").get("rules")]
  if rule_name in rules:
    permission_url += "/" + rule_name
    find = True
```

I will raise a PR for you to review.

Many thanks

**Comment by Turall** [comment]
Hi @migster8 , thank you for your interest in this project. I will merge this PR.

**Comment by Turall** [comment]
Released new version 1.3.4
